### DISCUSSION TITLE
How to fix the Gender Ratio?

### DISCUSSION THREAD

**Discussion by WilliamDormer** [PullDating/Org#130] [Pain Points]
In traditional dating apps, the gender imbalance can get up to 70% male. This is a risk that we want to avoid if we want our application to be effective. Particularly we need to be careful not to make our features all positive for men, and not attractive for women.

**Comment by barvian** [comment]
22_cobras [suggested on our Reddit post ](https://www.reddit.com/r/dating/comments/uuey9y/comment/i9fd2sb/?utm_source=reddit&utm_medium=web2x&context=3) to "LET PEOPLE SEE WHO LIKES YOU WITHOUT PAYING!" I think this could be an appealing feature for women (who get more likes on average) if it's possible to pull off financially. I'm not aware of a single dating app that offers it for free, and it's usually the first thing they advertise as part of their premium plan. Obviously it's not relevant if we conceal received likes, though.
